src/pybkgmodel/model.py

```diff
@@ -183,4 +204,83 @@ def get_runwise_bkg(self, target_run)->RectangularCameraImage:
         counts = numpy.sum([im.counts for im in images], axis=0)
         exposure = u.Quantity([im.exposure for im in images]).sum(axis=0)
 
+        return RectangularCameraImage(counts, self.xedges, self.yedges, self.energy_edges, exposure=exposure)
+    
+class OffDataMap:
+    """_summary_
+
+    Attributes
+    ----------
+    off_runs : tuple
+        Off data runs.
+    x_edges : numpy.ndarray
+        Array of the bin edges along the x/azimuth axis; linear binning.
+    y_edges : numpy.ndarray
+        Array of the bin edges along the y/Zenith axis; linear binning.
+    e_edges : numpy.ndarray
+        Array of the bin edges in energy; logarithmic binning.
+    regions : _type_
+        _description_
+    cuts : str
+        Event selection cuts.
+    pointing_delta : astropy.units.quantity.Quantity, optional
+        Pointing difference between runs for run matchin, by default 2*u.deg
+    """
+    def __init__(self, off_runs, x_edges, y_edges, e_edges, regions, cuts, pointing_delta=2*u.deg):
+        """_summary_
+
+        Parameters
+        ----------
+        off_runs : tuple
+            Off data runs.
+        x_edges : numpy.ndarray
+            Array of the bin edges along the x/azimuth axis; linear binning.
+        y_edges : numpy.ndarray
+            Array of the bin edges along the y/Zenith axis; linear binning.
+        e_edges : numpy.ndarray
+            Array of the bin edges in energy; logarithmic binning.
+        regions : _type_
+            _description_
+        cuts : str
+            Event selection cuts.
+        pointing_delta : astropy.units.quantity.Quantity, optional
+            Pointing difference between runs for run matchin, by default 2*u.deg
+        """
+        self.off_runs       = off_runs
+        self.xedges         = x_edges
+        self.yedges         = y_edges
+        self.energy_edges   = e_edges
+        self.regions        = regions
+        self.cuts           = cuts
+        self.pointing_delta = pointing_delta
+
+    def get_runwise_bkg(self, target_run)->RectangularCameraImage:
+        """_summary_
+
+        Parameters
+        ----------
+        target_run : _type_
+            _description_
+
+        Returns
+        -------
+        RectangularCameraImage
+            _description_
+        """
+        neighbours = find_run_neighbours(target_run, self.off_runs, self.pointing_delta)
```

Comment:
The non-optional `time_delta` argument is missing. The function signature is
```python
def find_run_neighbours(target_run, run_list, time_delta, pointing_delta)
```
I suppose some large value e.g. `time_delta = numpy.inf * u.hr` can be provided here, if I correctly understand the logic of the proposed `OffDataMap` class.